src/GenAPI/Microsoft.DotNet.GenAPI.Task/GenAPITask.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Build.Framework;
+using Microsoft.DotNet.Build.Tasks;
+using Microsoft.DotNet.GenAPI.Shared;
+
+namespace Microsoft.DotNet.GenAPI.Tasks;
+
+#nullable enable
```

Comment:
Turn on `<Nullable>enable</Nullable>` for the entire project and remove this line.